src/io.rs

```diff
@@ -2,85 +2,105 @@
 
 use core::{fmt, slice};
 use core::fmt::Write;
-
-/// File descriptors
-const STDOUT: usize = 1;
-const STDERR: usize = 2;
+use nr;
 
 /// Host's standard error
-struct Stderr;
+pub struct Stderr {
+    fd: usize,
+}
+
+/// Construct a new handle to the host's standard error.
+pub fn stderr() -> Stderr {
+    Stderr { fd: open(":tt\0", nr::open::W_APPEND).unwrap() }
```

Comment:
Is it documented anywhere that W_APPEND is used for stderr and W_TRUNC is used for stdout? If so could you add a link to that documentation?